app/posts/manage-school-placements/2024-02-23-beta-location-analysis-phase-4.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Beta location analysis - phase 4 - Understanding Leeds
+description: Important considerations for performance analysis of Leeds as the beta sample location
+date: 2024-02-23T12:03:00+00:00
+tags:
+  - data
+  - school placements
+related:
+  items:
+    - text: Beta location analysis - phase 1
+      href: /manage-school-placements/beta-location-analysis-phase-1/
+    - text: Beta location analysis - phase 2
+      href: /manage-school-placements/beta-location-analysis-phase-2/
+    - text: Beta location analysis - phase 3
+      href: /manage-school-placements/beta-location-analysis-phase-3/
+---
+
+
+
```

Comment:
Are providers alright with us sharing this information as they are named here - In other tables they are part of aggregate statistics so this is less of a concern?

Reply:
We decided to change the numbers to bands so we don't show actual numbers. The bands are:

- 1 to 25
- 25 to 50
- 51 to 75
- 76 to 100
- over 100